scripts/rollup/modules.js

```diff
@@ -44,9 +44,7 @@ function getPeerGlobals(externals, moduleType) {
 
 // Determines node_modules packages that are safe to assume will exist.
 function getDependencies(bundleType, entry) {
-  const packageJson = require(path.basename(
-    path.dirname(require.resolve(entry))
```

Comment:
Lint fails because we don't use `path` now. You can run `yarn linc` to lint changed files.